apps/envited-x-data-space/common/asset/utils.ts

```diff
@@ -299,6 +300,8 @@ export const extractGeneralInformationFromMetadata = (type: string) =>
     version: path([`${type}:hasDataResourceExtension`, `${type}:hasFormat`, `${type}:version`, '@value']),
   })
 
+export const extractManifestReferencedArtifacts = prop('manifest:hasReferencedArtifacts')
```

Comment:
Maybe `extractReferencedArtifacts` would be enough (since you feed it the manifest) but otherwise `extractReferencedArtifactsFromManifest`?